Migration to netcore2 + support for new project files

killwort opened the pull request:
Migrated to netcore 2.0-preview.
msbuild project support instead of obsoleted project.json.
Removed support for -t/-c arguments (this functionality should be implemented in the calling script).

killwort committed:
Migrated to netcore 2.0-preview.
msbuild project support instead of obsoleted project.json.
Removed support for -t/-c arguments (this functionality should be implemented in the calling script).

killwort commented:
I've pushed this fork to nuget with name dotnet-version-bump. If you'd like, you may push a new version with old name to maintain compatibility.

BalassaMarton commented:
@killwort Thank you for the PR, however I don't work on this project any more, I use my new tool [MSBump](https://github.com/BalassaMarton/MSBump) instead.

killwort committed:
Removed dependecies on nuget libs, cleaned up stdout to have parsable format (unrelated stuff moved to stderr)

UmmarIqbal committed:
- dotnet version updated to 6.0
- Command line arguments made pretty
- Solution option added, all project files are updated within a solution
- AssemblyVersion, FileVersion, Version, all three parameters are now updated

UmmarIqbal committed:
Docker file added + some fixes

UmmarIqbal committed:
argument fix

UmmarIqbal committed:
Readme updated

killwort committed:
Merge pull request #1 from ummarbhutta/master

Upgrade to dotnet 6.0 and solution file support

Commit:
Multitarget

UmmarIqbal committed:
- --csproj issue fixed
- --suffix option is added

ummarbhutta committed:
Merge branch 'killwort:master' into master

UmmarIqbal committed:
version fix

UmmarIqbal committed:
targeting only dotnet 6.0

UmmarIqbal committed:
dotnet 6.0 as a target

killwort committed:
Merge pull request #2 from ummarbhutta/master

bug fix and suffix support

Commit:
Version bump
